src/modules/auth/resources/ResourcesDeletedListContainer.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { DataTable, ButtonWithAuthz } from 'asab-webui';
+import { Container, Button } from 'reactstrap';
+
+const ResourcesDeletedListContainer = (props) => {
+
+	let SeaCatAuthAPI = props.app.axiosCreate('seacat_auth');
+
+	const [resources, setResources] = useState([]);
+	const [count, setCount] = useState(0);
+	const [page, setPage] = useState(1);
+	const [loading, setLoading] = useState(true);
+	const [show, setShow] = useState(false);
```

Comment:
@petrKavulok I think this is not needed. I think its ok if the content loader appears only for a second. You are adding jsut too much logic to this considering we dont use it elsewhere. But if you want to do it this way, then we should update it in other parts of app.

Reply:
In my subjective point of view, I think its enough to handle it just with `loading` state